x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/set_chrome.tsx

```diff
@@ -113,5 +114,22 @@ export const SetWorkplaceSearchChrome: React.FC<SetChromeProps> = ({ trail = []
   return null;
 };
 
+export const SetEnterpriseSearchContentChrome: React.FC<SetChromeProps> = ({ trail = [] }) => {
```

Comment:
This looks like the fifth function in this file that does the exact same thing, the only difference is two functions each uses: the `*Title` and `use*Breadcrumbs` functions... Should we DRY this up and create one function that accepts those two functions as arguments that we can reuse?

Reply:
Valid point. But I would keep it as is for now since this is not a big file yet and we are kind of using react components in a "different" way :D We are setting data but not rendering anything to simplify our code elsewhere. 

Totally fine with DRYing it out but it is OK as is too.

Reply:
I could maybe see abstracting a base component that is then used by the app specific components here. But have the app specific components here removes some complexity from the `page_template`s.

I've been bitten by DRY adding complexity enough times in the past, that I tend to not reach for it right away anymore.